common/lib/transport/connectionmanager.js

```diff
@@ -66,6 +68,8 @@ var ConnectionManager = (function() {
 			params.echo = 'false';
 		if(this.format !== undefined)
 			params.format = this.format;
+		if(this.stream !== undefined)
```

Comment:
All of this special-case crap for one of the transport params was what I was trying to get away from with the previous commit. The idea is that the stream param is added to params along with all other given transport params. The only special-casing needed is to take notice of whether or not it's a stream when creating the request.